recipes/rotor/all/conanfile.py

```diff
@@ -0,0 +1,109 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.43.0"
+
+
+class RotorConan(ConanFile):
+    name = "rotor"
+    license = "MIT"
+    homepage = "https://github.com/basiliscos/cpp-rotor"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = (
+        "Event loop friendly C++ actor micro-framework, supervisable"
+    )
+    topics = ("concurrency", "actor-framework", "actors", "actor-model", "erlang", "supervising", "supervisor")
+
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "fPIC": [True, False],
+        "boost_asio": [True, False],
+        "thread": [True, False],
+    }
+    default_options = {
+        "fPIC": True,
+        "boost_asio": True,
+        "thread": True,
+    }
+
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake", "cmake_find_package"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("boost/1.69.0")
```

Comment:
should this not be depended on the asio option?

Reply:
No. The project mandatory needs boost headers, datetime and system